js/utils.js

```diff
@@ -415,6 +415,162 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     }
   }
 
+  function getDateModifiedValues(options) {
+    var isEntryContainsTime = /^([0-1][0-9]|[2][0-3]):([0-5][0-9])$/.test(options.date);
```

Comment:
1. Set up `timestamp` variable to simplify the code below.
2. Based on your regex, it's not just that the timestamp "contains time", it contains "only" time. So, use a variable name that's more specific to its purpose.

```suggestion
    var timestamp = options.date;
    var timeOnly = /^([0-1][0-9]|[2][0-3]):([0-5][0-9])$/.test(timestamp);
```